src/ui/components/shared/LoadingTip.tsx

```diff
@@ -0,0 +1,82 @@
+import classNames from "classnames";
+import React, { useEffect, useRef, useState } from "react";
+import { connect, ConnectedProps } from "react-redux";
+import { actions } from "ui/actions";
+import { selectors } from "ui/reducers";
+import { UIState } from "ui/state";
+
+const TIPS = [
+  {
+    title: "SAML / SSO Integration",
+    description:
+      "Replay supports single sign-on via Google SAML 2.0 by default. Reach out if you would like to provision your SSO.",
+    icon: "zypsy-security-icon1.svg",
+  },
+  {
+    title: "Encryption at Rest & In-Transit",
+    description:
+      "Replay uses TLS v1.2 in transit and AES 256 at rest so that your data is secure at all times.",
+    icon: "zypsy-security-icon2.svg",
+  },
+  {
+    title: "Intellectual property",
+    description:
+      "We respect the privacy of everyone who uses our software and do not sell customer data.",
```

Comment:
It'd be nice to be able to demonstrate how folks source code and trade secrets are safe with us

> We architected our backend with source code, trade secrets, and personally identifiable information in mind.

Reply:
@jaril here are some new versions
https://www.notion.so/replayio/Security-tips-a77cc563fb4f4b1ca1cfd9b566b0a9f1